### DISCUSSION TITLE
Error extracting files

### DISCUSSION THREAD

**Discussion by WanderingMindTravels** [lllyasviel/Fooocus#2526] [Q&A] [answered]
While extracting the 7Z file from the download link using 7Zip, I got the following errors:

> Cannot open output file : Access is denied. : C:\run.bat
> Cannot open output file : Access is denied. : C:\run_anime.bat
> Cannot open output file : Access is denied. : C:\run_realistic.bat

How can I either get those files or fix the error? I've tried deleting the Fooocus folder and re-extracting, but I get the same error.

**Comment by mashb1t** [comment, marked as answer]
For some users it helped to extract the file with another tool, but i assume you're using the official 7z, right?
You can also open the files in a text editor (or right-click > edit) and copy the command to a terminal (non-admin in same folder) to run Fooocus manually.

**Comment by WanderingMindTravels** [comment]
I was using the official 7z. It worked fine when I used a different extraction app as you suggested. Thanks!
